helm-charts/templates/cert-manager/issuers/apk-root-certificate-secret.yaml

```diff
@@ -0,0 +1,12 @@
+{{ if and .Values.certmanager .Values.certmanager.enableRootCa }}
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: apk-root-certificate
+  namespace: {{ .Release.Namespace }}
+type: kubernetes.io/tls
+data:
+  tls.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUMvVENDQWVXZ0F3SUJBZ0lVZDRuanY4eVNQZ283dDBGMWUyYUpFbzlUcFE0d0RRWUpLb1pJaHZjTkFRRUwKQlFBd0RqRU1NQW9HQTFVRUF3d0RZWEJyTUI0WERUSXpNRE13T1RBNE1UWXpORm9YRFRNek1ETXdOakE0TVRZegpORm93RGpFTU1Bb0dBMVVFQXd3RFlYQnJNSUlCSWpBTkJna3Foa2lHOXcwQkFRRUZBQU9DQVE4QU1JSUJDZ0tDCkFRRUExRzlCQWNWYTNhTWVteEVSWStKOFVtSC9XOEpwY0VPdGRlTlg1WU5paG5OT1h0bHZoU3pGek9QYUs5N1AKcjRJcUdBU2JWTmlSKzFKNldFb2kvYjZaSlR4MHEzWVVuMFlsUUpyejdnMjBUZG9HSmp4R1ZXem4wRVc0YmVIWApHcTYwdlhMZjR0M21sTENMR0lLM2tKVFdBb1J6ZDc0ZGpWNys1djBCbS82S0JCQVdjdTVVYk9EOUtScE9zeEdNCm4zWjAxMDNvQUdWaXlxODRRdEZ2aFhWTld0dERMZTJqVS83bzQyZGRhSm96Ukw5eisxQWVwZG9XUHlKWklacVUKYlhjR0FrN2lkazdjLzhkS014d0FtM0NWL1d2Z1dyVks1UitZVGlHcVJmNXBkOVdXQ3lkRVZRa05xQ1pnVFBOeQpCVFJ2SG81Mm9uUG5UNkFMdE1JMG1uV0x0UUlEQVFBQm8xTXdVVEFkQmdOVkhRNEVGZ1FVenhjQThjZUNGNXQrCnZQZU9wWWJpMTFDV2p3Y3dId1lEVlIwakJCZ3dGb0FVenhjQThjZUNGNXQrdlBlT3BZYmkxMUNXandjd0R3WUQKVlIwVEFRSC9CQVV3QXdFQi96QU5CZ2txaGtpRzl3MEJBUXNGQUFPQ0FRRUFGQ0tkNXg4ejY0cDFqOUJub1NBbApKSnVOejR1V2p2L1lLOTRCM3MzS1VrWE9Yakh1THVhemtTc3MyVUhGYitLZUxkNnZoaW1NNVFxa0ZzVUVNc25DCjhSeUJaUDU4a2doY0x6SmxkNVVpd2xuci9SdU9BTnZjdjRlS1Nhdnd1NS9BQlh1TVVRYi9Hdkt0V1BycjJWYkoKVUxnM3A3TkdpZ1hISGc4NGVWTUE3b05YMVo1UjJjUzRJU2tsV1htNVNwTVBoK1NOQ2dxd3FoeFJOWUoySjBFWgpxbHA0b2ZRRzNHSjcySitEUkhsTnVqV0Vza1A1SUpqdzZ3OFEwempYeDI2eWVsR2UyK1RNNkJCN1BwQ042a05VCnpIbzJrLzU3NWJ1MmlaenRuWVZtRTc0SDFXM2NYSjdjMHE4MnVVRnZkVytGbFJ0bStPUElpSUdLNzRsRmlaTkIKT1E9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

Comment:
shall we add other issuer properties for this CA considering wso2 org properties?
```
    "issuer": {
        "C": "US",
        "ST": "CA",
        "L": "Mountain View",
        "O": "WSO2",
        "OU": "WSO2",
    }
```